asset/cloud-client/quickstart_exportassets_test.py

```diff
@@ -44,9 +51,35 @@ def asset_bucket(storage_client):
         raise e
 
 
-def test_export_assets(asset_bucket, capsys):
+@pytest.fixture(scope='module')
+def dataset(bigquery_client):
+    dataset_id = "{}.{}".format(PROJECT, DATASET)
+    dataset = bigquery.Dataset(dataset_id)
+    dataset.location = "US"
+    try:
+        dataset = bigquery_client.create_dataset(dataset)
+    except Exception as e:
+        print("Failed to Create dataset {}".format(dataset_id))
+        raise e
+
+    yield DATASET
+
+    try:
+        bigquery_client.delete_dataset(dataset_id, delete_contents=True,
```

Comment:
Nit: This formatting looks a bit strange. Technically allowed by the Google style guide, but inconsistent with what `black` does (which would break after `(`)